src/live.asp.net/Views/Shared/_Layout.cshtml

```diff
@@ -59,6 +57,8 @@
     @RenderSection("PostContent", required: false)
 
     @RenderSection("scripts", required: false)
+ <partial name="_AnalyticsHead" />
```

Comment:
If the partial moved to the bottom of the body, why we need two instead of one named "Analytics"